followthemoney/schema/CallForTenders.yaml

```diff
@@ -0,0 +1,77 @@
+CallForTenders:
+  label: "Call for tenders"
+  plural: "Calls for tenders"
+  description: "A public appeal issued by an authority, possibly on behalf of another, for buying a specific work, supply or service"
+  extends:
+    - Thing
+  required:
+    - title
+  featured:
+    - title
+    - contractingAuthority
+  caption:
+    - title
+  properties:
+    title:
+      label: "Title"
+    uid:
+      label: "CfT Unique ID"
+    cftCaUid:
+      label: "CfT CA Unique ID"
+    contractingOnBehalfOf:
+      label: "Published on behalf of"
```

Comment:
Should this be an entity reference to a LegalEntity?

Reply:
It should eventually yes, but I'd like to make a flat version first, and when we agree on the whole model and I get the process for emitting it from a memorious scraper, I'll make a structured one if you may accept.